doc/release-notes.md

```diff
@@ -50,33 +53,246 @@ is not recommended.
+1152-1440 block range after the activation. Any nodes that do not upgrade by
+that time will diverge from the rest of the network.
+
+Deterministic InstantSend
+-------------------------
+Deterministically verifying InstantSend locks at any point in the future has
+been added to support Dash Platform. This update introduces versioning to
+InstantSend messages and adds quorum information to them. While the previous
+design was sufficient for core chain payments, the platform chain will benefit
+from this enhanced verification capability. Details about deterministic
+InstantSend are provided in [DIP-0022](https://github.com/dashpay/dips/blob/master/dip-0022.md).
+
+Deterministic InstantSend will be activated with the DIP0024 hard fork.
+
+Governance
+----------
+Several improvements have been made to Dash’s DAO governance system.
+The governance proposal fee has been reduced from 5 Dash to 1 Dash following
+a vote by masternode owners to do so. For improved security and flexibility,
+proposal payouts to pay-to-script-hash (P2SH) addresses are now supported.
+
+These changes wil be activated with the DIP0024 hard fork.
+
+Governance proposals can now be viewed in GUI Governance tab (must be enabled
+in Preferences first).
+
+Initial Enhanced Hard Fork support
+----------------------------------
+The masternode hard fork signal special transaction has been added as the first
+step in enabling an improved hard fork mechanism. This enhancement enables
+future hard forks to be activated quickly and safely without any
+“race conditions” if miners and masternodes update at significantly different
+speeds. Effectively there will be a masternode signal on chain in addition to
+the miner one to ensure smooth transitions. Details of the enhanced hard fork
+system are provided in [DIP-0023](https://github.com/dashpay/dips/blob/master/dip-0023.md).
+
+Network improvements
+--------------------
+We implemented and backported implementations of several improvement proposals.
+You can read more about implemented changes in the following documents:
+- [`DIP-0025`](https://github.com/dashpay/dips/blob/master/dip-0025.md): Compressed headers.
+- [`BIP 155`](https://github.com/bitcoin/bips/blob/master/bip-0155.mediawiki): The 'addrv2' and 'sendaddrv2' messages which enable relay of Tor V3 addresses (and other networks).
+- [`BIP 158`](https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki): Compact Block Filters for Light Clients.
+
+KeePass support removed
+-----------------------
+Please make sure to move your coins to a wallet with a regular passphrase.
+
+Wallet changes
+--------------
+We continued backporting wallet functionality updates. Most notable changes
+are:
+- Added support for empty, encrypted-on-creation and watch-only wallets.
+- Wallets can be created, opened and closed via a GUI menu now.
+- No more `salvagewallet` option in cmd-line and Repair tab in GUI. Check the
+`salvage` command in the `dash-wallet` tool.
```

Comment:
```suggestion
- No more `salvagewallet` option in cmd-line and Repair tab in GUI. Check the
`salvage` command in the `dash-wallet` tool.
- Creating receive addresses in the GUI must now be done using the "Receive" tab
```

Reply:
Hasn't this been the case since before this version?

Reply:
I don't think so :thinking:

Reply:
Oh, I guess so. Disregard this comment

Reply:
I can confirm that there is no "Create receiving address" button (or whatever label it had earlier) in v0.17.0.3